### PR TITLE
Result percentage changed to a fraction

### PR DESCRIPTION
**Pull request by MasloMaslane**
Sioworkers will (https://github.com/sio2project/sioworkers/pull/24) return `result_percentage` as a tuple representing a fraction. The tests will fail until there is a new release of sioworkers with the change.

### CODE REVIEW

File: oioioi/programs/utils.py [hunk 1]
```diff
@@ -98,8 +98,8 @@ def min_group_scorer(test_results):
 
 def discrete_test_scorer(test, result):
     status = result['result_code']
-    percentage = result.get('result_percentage', 100)
-    max_score = int(ceil(percentage * test['max_score'] / 100.))
+    percentage = result.get('result_percentage', (100, 1))
+    max_score = int(ceil(percentage[0] * test['max_score'] / (100. * percentage[1])))
```
Comment (twalen) [inline oioioi/programs/utils.py:102]: what about using Fraction and `math.ceil(Fraction(pr) * max_score )`

**Comment by twalen** [comment]
the build is still failing

**Comment by MasloMaslane** [comment]
I fixed the tests

**Review by twalen — APPROVED** [review]
OK

**Pull request merged**